### PR TITLE
(#1642728) udev: make algorithm that selects highest priority devlink less susceptible to race conditions

### PR DESCRIPTION
**Pull request by msekletar**

### CODE REVIEW

**Comment by msekletar** [comment]
Travis CI failure is due to Dockerhub rate-limiting and hence inability of the setup script to set up CentOS 8 container on Ubuntu.

File: test/udev-test.pl [hunk 1]
```diff
@@ -18,11 +18,19 @@
```
Comment (dtardon) [inline test/udev-test.pl]: Do we really need this commit?

Comment (dtardon) [reply test/udev-test.pl]: Well, it doesn't matter that much...

**Review by dtardon — APPROVED** [review]
LGTM

**Pull request merged**